build.gradle.kts

```diff
@@ -55,7 +55,7 @@ tasks.jacocoTestReport {
     dependsOn(tasks.test) // tests are required to run before generating the report
     reports {
         xml.required.set(true)
-        csv.required.set(false)
-        html.required.set(false)
+        csv.required.set(true)
+        html.required.set(true)
```

Comment:
Why do we need all those tests? Just to have more options?

Reply:
We don't really need all those, I just turned them on because it's the default settings